apps/server/lib/lexical/server/code_intelligence/completion/translations/struct.ex

```diff
@@ -20,6 +19,19 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Struct do
     end
   end
 
+  def completion(%Env{} = env, builder, module_name, full_name, more) when is_integer(more) do
+    builder_opts = [
+      kind: :module,
+      label: "#{module_name}...(#{more} more structs)",
+      detail: "#{full_name}."
+    ]
+
+    insert_text = "#{module_name}."
+    range = edit_range(env)
+
+    builder.text_edit_snippet(env, insert_text, range, builder_opts)
+  end
```

Comment:
I really don't like this pattern, translations are about translating an elixir sense completion into LSP. What we want here is to collapse and aggregate module completions that are returned from remote_control. Do we need another place to do this work?